Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=22c733788bbd4b75c00279119a83da5cd74b987a
Commit:     22c733788bbd4b75c00279119a83da5cd74b987a
Parent:     1de9e8e70f5acc441550ca75433563d91b269bbe
Author:     Clemens Ladisch <clemens@ladisch.de>
AuthorDate: Mon Jan 19 10:07:21 2009 +0100
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Mon Jan 19 14:07:21 2009 +0100

    sound: virtuoso: enable UART on Xonar HDAV1.3
    
    This hardware has a better chance of working correctly if we don't
    forget to enable it.
    
    Signed-off-by: Clemens Ladisch <clemens@ladisch.de>
    Cc: <stable@kernel.org>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

 sound/pci/oxygen/virtuoso.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/sound/pci/oxygen/virtuoso.c b/sound/pci/oxygen/virtuoso.c
index e9e829e..a962166 100644
--- a/sound/pci/oxygen/virtuoso.c
+++ b/sound/pci/oxygen/virtuoso.c
@@ -908,6 +908,7 @@ static const struct oxygen_model model_xonar_hdav = {
 	.dac_channels = 8,
 	.dac_volume_min = 0x0f,
 	.dac_volume_max = 0xff,
+	.misc_flags = OXYGEN_MISC_MIDI,
 	.function_flags = OXYGEN_FUNCTION_2WIRE,
 	.dac_i2s_format = OXYGEN_I2S_FORMAT_LJUST,
 	.adc_i2s_format = OXYGEN_I2S_FORMAT_LJUST,
